inspire/base/templates/search/results.html

```diff
@@ -28,6 +28,55 @@
 {%- block search_result_settings_display %}
 {%- endblock search_result_settings_display %}
 
+{% block search_result_nb_recs scoped %}
```

Comment:
I think we should overwrite everything from the parent template and remove the wrapping

```
<form class="form-inline" name="" action="{{ url_for('search.dispatch') }}" method="post">
```

as we don't user that `endpoint` and it can avoid problems.